### ISSUE TITLE
HTTP response is not recorded when the responseType is not JSON

### ISSUE THREAD

**Issue by Nanciee** [Nanciee/cypress-autorecord#7]
one last qq(sorry): My responses are always empty objects, is that intended?
```
{
  "user can login successfully": [
    {
      "url": "http://localhost:5100/web-api/v1/size-preferences",
      "method": "GET",
      "status": 200,
      "body": null,
      "response": {}
    }
  ]
}
```

looks like the `response.response.body` is of type Blob when I log it 
![image](https://user-images.githubusercontent.com/3660667/60745941-546aa580-9f31-11e9-963f-16b1eadfb2b4.png)

any idea how to get the actual response?

_Originally posted by @bautistaaa in https://github.com/Nanciee/cypress-autorecord/issues/5#issuecomment-508857062_

**Comment by manphannvg** [comment]
Comment because I care for this issue. I also got the same problem

**Comment by lcnandre** [comment]
Just created PR #33 to solve this. Please take a look at the PR comments.

**Comment by Nanciee** [comment]
Resolved in [#33 ] has been merged, closing this issue
